docs/_articles/en/user-guide/byotemplate.md

```diff
@@ -4,79 +4,34 @@ lang: en
 ---
 
 ## Overview
-Want your own code to appear in source files when you create a metadata object such as an Apex class? You can now use custom templates to do just that.
+Use custom templates to quickly add your own code template to source files when you create a metadata object such as an Apex class in VS Code. Templates are files that contain your custom code. You can use custom templates to -
+- Add default copyright information to new files in your project.
+- Add default code to Aura or LWC enable a new Apex class. etc.
 
-Templates are essentially folders with files that contain your custom code. This [git repo](https://github.com/forcedotcom/salesforcedx-templates/tree/main/src/templates) contains a collection of official Salesforce templates for metadata components. Simply clone this repo, keeping the same folder structure, then update relevant template files with your code. Remove the files that you don’t wish to override.
-
-**Note:** Only updates made to the files listed here show up in source files in VS Code. There’s no such restriction when you use the CLI to specify templates for metadata objects.
```

Comment:
I'll confirm that we have this info in the doc already but i am not a fan of keeping that table...

Reply:
Sounds good - agreed the table is not the prettiest of options! It is helpful because there are extra files at times, like for apexclass there is ApexUnitTest, but you can't use that with VS Code, only with the CLI.